src/renderers/webgl/WebGLState.js

```diff
@@ -336,6 +336,10 @@ function WebGLState( gl, extensions, paramThreeToGL ) {
 
 	var maxTextures = gl.getParameter( gl.MAX_TEXTURE_IMAGE_UNITS );
 
+	var glVersion = gl.getParameter(gl.VERSION).split(' ')[1];
```

Comment:
As said before, WebGL future version might be tagged like "WebGL 2.0.3", I will just modify the regex a bit and that would work fine ;)